packages/core/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+All notable changes to the Zowe core SDK package will be documented in this file.
+
+## `1.0.0`
```

Comment:
is core going to be independently versioned?

Reply:
No, they're supposed to be versioned together.